ui/src/app/machines/hooks.tsx

```diff
@@ -44,9 +46,24 @@ export const useMachineActionForm = (
   // If in the machine details view, the machine is not in selected state so
   // instead we use the regular selector.
   const processingMachines = useSelector(
-    machineSelectors[activeMachine ? action.status : `${action.status}Selected`]
+    machineSelectors[
+      selectedMachines ? action.status : `${action.status}Selected`
```

Comment:
Looks like the logic has flipped here and should be flipped back.

Reply:
Strange, done.